src/object/obj_tx.c

```diff
@@ -2139,11 +2140,6 @@ dc_tx_local_open(daos_handle_t coh, daos_epoch_t epoch, uint32_t flags,
 	struct dc_tx	*tx = NULL;
 	int		 rc;
 
-	if (epoch == 0 || epoch == DAOS_EPOCH_MAX) {
-		D_ERROR("Invalid epoch for tx_local_open\n");
-		return -DER_INVAL;
-	}
-
```

Comment:
ah, it was for the usage in tx_local because it pass epoch 0, as I removed in my last version, then this check need not remove, let me refresh it to add it back.

Reply:
It is OK for me the re-add them back when have other chance to refresh the patch.